client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -15,11 +15,31 @@ varying vec3 tsEyeVec;
 varying vec3 lightVec;
 varying vec3 tsLightVec;
 
-bool normalTexturePresent = false; 
+bool normalTexturePresent = false;
+bool texTileableHorizontal = false;
+bool texTileableVertical = false;
+bool texSeamless = false;
 
 const float e = 2.718281828459;
 const float BS = 10.0;
- 
+
+void getTextureFlags ()
+{
+	vec4 flags = texture2D(textureFlags, vec2(0.0, 0.0));
+	if (flags.r > 0.5) {
```

Comment:
You have single line if statements surrounded by brackets, which is okay on its own, but you keep switching between having brackets and not having brackets within your own code.  Please, choose one style and stay consistent!